homework/hackyourtemperature/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "hackyourtemperature",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "node server.js",
```

Probably should switch this to:
```suggestion
    "start": "node app.js",
```

since you moved your app file 😄